Integrations/integration-PhishMe_CHANGELOG.md

```diff
@@ -1,2 +1,5 @@
 ## [Unreleased]
+
+
+## [19.8.0] - 2019-08-06
 Deprecated. Use the Cofense Intelligence integration instead.
```

Comment:
Do we curently know how to treat deprecated integrations? in last RN had to due it manually